pkg/kubelet/apis/config/v1beta1/BUILD

```diff
@@ -20,6 +21,7 @@ go_library(
     importpath = "k8s.io/kubernetes/pkg/kubelet/apis/config/v1beta1",
     deps = [
         "//pkg/kubelet/apis/config:go_default_library",
+        "//pkg/kubelet/apis/config/constant:go_default_library",
```

Comment:
I'm confused here, because the right path for `constant.go` is going to be `pkg/kubelet/apis/config/v1beta1/constant.go`. but if I execute the `./hack/update-bazel.sh` script, the `v1beta1` will be removed from the path.

Reply:
in master with the head commit 990695c839a99cbd0e00fd3806f9802e2af7c2e0:

does constant.go will **be generated** into `pkg/kubelet/apis/config/v1beta1/`?
and i can only `find` a constant.go under `cmd/kubeadm/app/cmd/options/`.

btw, what's the meaning of [this](https://github.com/kubernetes/kubernetes/blob/master/hack/lib/init.sh#L63)?
only if one is within `KUBE_AVAILABLE_GROUP_VERSIONS`, it will be treated?
if so, why `config/v1beta1` is not in?